src/annotator/test/guest-test.js

```diff
@@ -22,8 +22,8 @@ class FakeAdder {
 FakeAdder.instance = null;
 
 class FakePlugin extends Plugin {
-  constructor() {
-    super();
+  constructor(element, config) {
+    super(element, config);
```

Comment:
Not forwarding the mandatory `element` argument here was a mistake that happened not to break anything previously.

Reply:
Neat!